app/models/manageiq/providers/ansible_tower/shared/automation_manager/configuration_script_source.rb

```diff
@@ -3,13 +3,24 @@ module ManageIQ::Providers::AnsibleTower::Shared::AutomationManager::Configurati
 
   module ClassMethods
     def create_in_provider(manager_id, params)
-      manager = ExtManagementSystem.find(manager_id)
-      project = manager.with_provider_connection do |connection|
-        connection.api.projects.create!(params)
-      end
+      begin
+        manager = ExtManagementSystem.find(manager_id)
+        project = manager.with_provider_connection do |connection|
+          connection.api.projects.create!(params)
+        end
 
-      refresh(manager)
-      find_by!(:manager_id => manager.id, :manager_ref => project.id)
+        refresh(manager)
+        record = find_by!(:manager_id => manager.id, :manager_ref => project.id)
+      rescue AnsibleTowerClient::ClientError => tower_error
+        _log.error "create_in_provider failed: #{tower_error}"
```

Comment:
Also, you don't need to log this since ansible_tower_client will already log its failures to this log file.